packages/angular_devkit/build_optimizer/src/transforms/scrub-file.ts

```diff
@@ -13,6 +13,7 @@ export function testScrubFile(content: string) {
   const markers = [
     'decorators',
     '__decorate',
+    'tslib.__decorate',
```

Comment:
this is not sufficient marker because e.g. typescript 2.5 emits the import of tslib as:

```
import * as tslib_1 from "tslib";
```